servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpCookiePair.java

```diff
@@ -32,7 +36,7 @@
      * @param cookieName The <a href="https://tools.ietf.org/html/rfc6265#section-4.1.1">cookie-name</a>.
      * @param cookieValue The <a href="https://tools.ietf.org/html/rfc6265#section-4.1.1">cookie-value</a>.
      */
-    public DefaultHttpCookiePair(final CharSequence cookieName, final CharSequence cookieValue) {
+    public DefaultHttpCookiePair(final @Nonnull CharSequence cookieName, final @Nonnull CharSequence cookieValue) {
```

Comment:
The annotations are not needed here, at package level we keep all elements Nonnull by default (see. package-info.java)